_posts/2021-07-09-june-rubygems-updates.md

```diff
@@ -0,0 +1,40 @@
+---
+title: June 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in June.
+
+## RubyGems News
+
+This month in RubyGems, We released new versions for **Bundler `2.2.0`** and **RubyGems `3.2.20`**. The release for RubyGems included a security fix contributed by [Aditya](https://github.com/sonalkr132), and Bundler release ships with several fixes and improvements.
+
+We also worked on seamlessly improving the migration of insecure lock files by automatically dealing with them when possible instead of printing a warning and still installing them - [#4647](https://github.com/rubygems/rubygems/pull/4647) and [#4683](https://github.com/rubygems/rubygems/pull/4683).
+
+We suggested a course of action to take on - [#4031](https://github.com/rubygems/rubygems/issues/4031) since the inaction seems to be upsetting many people and investigated and created a test case for when Bundler install is stuck - [#4539](https://github.com/rubygems/rubygems/issues/4539).
```

Comment:
I would replace the above paragraph with

> We also kept discussing how to improve the experience when bundler needs sudo access, making the behaviour less upsetting for our users.

I would personally not mention the part about `bundle install` being stuck because that only led to a generic discussion about version resolvers and not much progress was made.

Reply:
Alright, will add this update. thanks!